azext_iot/tests/__init__.py

```diff
@@ -80,6 +80,13 @@ def __init__(self, test_scenario, entity_name, entity_rg):
         self.entity_rg = entity_rg
 
         super(IoTLiveScenarioTest, self).__init__(test_scenario)
+
+        self.cmd(
```

Comment:
This will considerably slow down inner dev loop when you are iterating on code and executing IT's against your changes, then having to wait each time for the long-running operation of creating an IoT Hub to finish. Ideally the tests harness can be flexible to use an existing target hub if desired, or provision a new one to run tests against.